### ISSUE TITLE
Html file doesn't show any mention (or way to see a conversation)

### ISSUE THREAD

**Issue by lapineige** [kensanata/mastodon-backup#36]
Hello,

(first of all, thanks a lot for this great tool ;)

I tried to download an archive with `--with-mentions`, it finished correctly and displayed [some plausible number] mentions.
But in the HTML file I don't see any mention. And I can't see a conversation, apart from opening the original web URL (and that's not the goal in archiving mentions).

(and I'm not very sure all mentions are here, I feel like it's a bit too small)

Do you know what's happening ?

**Comment by kensanata** [comment]
I think the HTML file only uses your posts. I probably felt that publishing what other people were writing to you was tricky and so I never implemented that. The mentions are there for you to search using the `text` command, and reports.

**Comment by lapineige** [comment]
So it's intended to be published ?
I'd like to do it to see the conversations (including media files ?), cause in text files it's not the most comfortable, and I feel it's handy to do a quick text or visual search in the web page.

**Comment by kensanata** [comment]
Well, I published [mine](https://alexschroeder.ch/mastodon.weaponvsac.space.user.kensanata.html) back then, and I never went back to the code. Somebody would have to add this...

**Comment by lapineige** [comment]
Ok, I understood it was only a nicer way to present the archive, not a way to publish it.

---

Ok I think I'll do it when I have a bit of time.
> There's also the issue of how to display conversations in static HTML files.

Good question, especially if we stick to a static file... As all toots are displayed in a vertical column, maybe it could show the answer horizontally ? (I'd also have to find a way to display a conversation with multiple branches clearly, not just one after another) 

I'd also have to add the avatars, to make that a bit nicer to read.

> I think it also only downloads media files of your own boosts, not of mentions, so somebody would have to add that as well.

Yeah I could add an option for that, but that's not my priority, we'll see :)
Additional question: if you download your archive after deleting some toots, are they removed ?

**Comment by kensanata** [comment]
>  if you download your archive after deleting some toots, are they removed ?
No. I use the tool to keep a copy of my toots and to delete my older toots. But I still want them in my archive.

**Comment by lapineige** [comment]
That's perfect. And it will ease my tests.
